dev-libs/libnv-portable/libnv-portable-1.0.1.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multibuild cmake
+
+DESCRIPTION="A userspace version of libnv ported from FreeBSD"
+HOMEPAGE="https://github.com/Zer0-One/libnv-portable"
+SRC_URI="https://github.com/Zer0-One/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="amd64"
+IUSE="static-libs test"
+
+DEPEND="${RDEPEND} test? ( dev-util/kyua dev-libs/atf )"
```

Comment:
This should be expanded for readability (and nicer diffs when comparing versions)
```ebuild
DEPEND="${RDEPEND}
 	test? (
 	 	dev-util/kyua
 	 	dev-libs/atf
 	)
"